tutorials/magento-ubuntu/index.mdx

```diff
@@ -0,0 +1,376 @@
+  </Directory>
+  ``` 
+## Setting up PHP
+
+As Magento has specific requirements for PHP, it is recommended to modify the PHP configuration file. 
+
+1. Open the file in a text editor.
+
+  ```
+  nano /etc/php/7.4/apache2/php.ini
+  ```
+
+2. Modify parameters as indicated below, save and exit.
+
+  ```sh no-copy
+  file_uploads = On
+  allow_url_fopen = On
+  short_open_tag = On
+  memory_limit = 4096M
+  cgi.fix_pathinfo = 0
+  opcache.save_comments = 1
+  upload_max_filesize = 150M
+  max_execution_time = 1800
+  realpath_cache_size=10M
+  realpath_cache_ttl=7200
+  date.timezone = Europe/Paris
+  ```
+3. Restart Apache.
+  ```
+  service apache2 restart 
+  ```
+
+## Setting up MariaDB
+
+1. Install the MariaDB database server and client using the apt package manager. MariaDB is an open-source fork of the MySQL relational database system and provides compatibility with all MySQL commands. Magento has been tested to comply with all MariaDB features.
+
+  ```
+  apt install mariadb-server mariadb-client
+  ```
+
+2. Initialize the MariaDB database server by running the following command:
+
+  ```
+  sudo mysql_secure_installation
+  ```
+
+    You will be prompted to answer the following questions during installation:
+
+    - **Enter current password for root (enter for none):** Press enter to continue, as the default MariaDB root password is empty.
+    - **Set root password? [Y/n]:** Type **Y** and press enter to continue.
+    - **New password:** Enter a secret password of your choice and press enter to continue.
+    - **Re-enter new password:** Repeat your password and press enter to confirm.
+    - **Remove anonymous users? [Y/n]:** Type **Y**, then press enter
+    - **Disallow root login remotely? [Y/n]:** Type **Y** and confirm by pressing enter.
+    - **Remove test database and access to it? [Y/n]:** Type **Y** and press enter to continue.
+    - **Reload privilege table now? [Y/n]:** Confirm by typing **Y**, then press enter.
+
+3. Now that the database engine is configured, log into MariaDB using the `root` user and enter the password set in the previous step when prompted:
+
+  ```
+  mysql
+  ```
+
+4. Create new database for magento by running the following command in the MariaDB shell:
+
+  ```
+  CREATE DATABASE magento;
+  ```
+
+5. Create a user for magento. Replace `your_secret_password` with a password for the user to authenticate:
+
+  ```
+  CREATE USER 'magento_db'@'localhost' IDENTIFIED BY 'your_secret_password';
+  ```
+
+6. Grant the newly created user all permissons on the `magento` database:
+
+  ```
+  GRANT ALL ON magento.* TO 'magento_db'@'localhost' IDENTIFIED BY 'your_secret_password' WITH GRANT OPTION;
+  ```
+
+7. Save the changes and quit the MariaDB shell:
+
+  ```
+  FLUSH PRIVILEGES;
+  EXIT;
+  ```
+
+## Downloading and installing Elasticsearch
+
+Since with its 2.4 version, Magento uses [Elasticsearch](https://www.elastic.co/) as the catalog search engine. Install it by following these steps:
```

Comment:
```suggestion
Since the 2.4 version of Magento has been released, it uses [Elasticsearch](https://www.elastic.co/) as the catalog search engine. Install it by following these steps:
```